virttest/libvirt_vm.py

```diff
@@ -2570,6 +2570,28 @@ def vcpuinfo(self):
                 vcpuinfo_list.append(vcpuinfo_dict)
         return vcpuinfo_list
 
+    def domfsinfo(self):
+        """
+        Return a dict's list include domain mounted filesystem information
+        via virsh command
+        """
+        result = virsh.domfsinfo(self.name,
+                                 uri=self.connect_uri).stdout.strip()
+        lines = result.splitlines()
+        domfsinfo_list = []
+        if len(lines) > 2:
+            head = lines[0]
+            lines = lines[2:]
+        else:
```

Comment:
remove line 2585 and 2586,and move code line 2588 to 2591 to block if len(lines) > 2:

Reply:
good point, thanks~